src/main/kotlin/racingcar/domain/numbergenerator/StubNumberGenerator.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.domain.numbergenerator
+
+class StubNumberGenerator(
+    private val fakeNumbers: MutableList<Int>,
+) : NumberGenerator {
+    override fun generate(): Int = fakeNumbers.removeAt(0)
```

Comment:
```suggestion
    override fun generate(): Int = fakeNumbers.drop(1)
```
drop 함수를 활용하셔도 좋습니다.

Reply:
drop은 list를 반환해서 Int를 반환하는 함수라 오류나는데 list를 반환하는것이 더 좋을까요?

Reply:
아 이부분은 제가 코드를 잘못 봤네요. 😅 
창환님이 구현하신 방향이 맞습니다. 🙏